themes/edx.org/lms/templates/header.html

```diff
@@ -72,7 +72,7 @@ <h2 class="course-header">
         % if not nav_hidden or show_program_listing:
           <nav aria-label="Main" class="nav-main">
             <ul class="left list-inline nav-global authenticated">
-              % if not nav_hidden:
+              % if is_account_settings_page is UNDEFINED and not nav_hidden:
```

Comment:
@schenedx alternatively we can do this: `% if is_account_settings_page is UNDEFINED and not (is_account_settings_page and nav_hidden):`